static/app/views/performance/transactionSummary/transactionEvents/index.tsx

```diff
@@ -181,10 +181,15 @@ function EventsContentWrapper(props: ChildProps) {
                 );
               }
 
-              const percentiles: PercentileValues = tableData
-                ?.data?.[0] as any as PercentileValues;
+              const percentileData = tableData?.data?.[0];
```

Comment:
oh I see what happened here. we've switched from `eventsv2` to `events` endpoint and they fields are returned differently `p50` vs `p50()` like here. can we check how many users are still on `eventsv2` because now this won't work for them

Reply:
i think the goal is to switch to using `/events` and deprecate `/eventv2` eventually but let's see how many users have migrated to using the new endpoint.
cc: @edwardgou-sentry would you know if we should support `eventsv2` here or if it's safe to just support `events`

Reply:
Interesting how were switching from `eventsv2` to `events`, would have assumed the other way around, curious on the differences between the two.

> can we check how many users are still on `eventsv2` because now this won't work for them
Does this mean `eventsv2` isn't deployed everywhere? 

I was assuming that they are just different request paths,
aka `/organizations/sentry/eventsv2/` vs `/organizations/sentry/events/` available on the same server, which means that the users would not be affected by this change and it should fix it, we just didn't complete all the code changes to work with `events`.

But if its the same path then that's a breaking change and we would have see how many users are using v2.

Reply:
> Interesting how were switching from eventsv2 to events, would have assumed the other way around, curious on the differences between the two.

there was `events` first, then it got deprecated and `eventsv2` was released. now, `eventsv2` will be deprecated and new `events` will be used. the difference between `eventsv2` and new `events` is minor AFAIK. like this is an example of how fields changed.

Reply:
That makes sense, thanks for the explanation!